src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -1096,7 +1179,96 @@ private void editCurrentlySelectedModElementAsCode(ModElement mu, JComponent com
 			ProjectFileOpener.openCodeFile(mcreator, modElementFiles.get(0));
 		}
 	}
+	
+        private void deleteCurrentlySelectedModElement() {
+		if (but3.isEnabled()) {
+			if (list.getSelectedValue() != null) {
+				Object[] options = { "Yes", "No" };
+				int n = JOptionPane.showOptionDialog(mcreator,

```

Comment:
Instead to use `JOptionPane#showOptionDialog`, `JOptionPane#showConfirmDialog` should allow you to use the default option pane of Java Swing (`Yes` and `No` are translated in the language of the user's computer).
You will also have to change `JOptionPane#QUESTION_MESSAGE` for `JOptionPane#YES_NO_OPTION`.

Reply:
This one is not something changed, i prefer not touch it,
As I said, the code was not something i written, i just copied it, and paste it in a method

Reply:
This isn't something i changed, i just copied the code, and pasted it in a method, so i can use it at 2 places
so i prefer not to change it